CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 
 # UNRELEASED
 
+- Sync snapshots directly to hotstore.  Save 120 G of lotus disk space. ([filecoin-project/lotus#12800](https://github.com/filecoin-project/lotus/pull/12803))
```

Comment:
As per my comments, "120 G" is overselling it by quite a bit, we need to nuance this and probably not even put a number on it.

Reply:
Technically correct @rvagg but this is not really the christmas spirit. 🎅  😢 

I really don't buy that this is significantly overselling it. Its just the difference in startup sizes immediately after syncing from snapshot and a lot of us do that pretty often.  Plus it looks like its also exactly the high water mark which matters because that's the value that matters when you need to get bigger disks.

I guess we should have a graph of hotstore size over time to quantify it.  After 8 months your hot store has leaned down as the state has replaced itself but its a slow process.

Reply:
And hey I said "~120" !

Reply:
I think you just missed the full copy of all cold objects to hot in the warmup procedure: https://github.com/filecoin-project/lotus/blob/master/blockstore/splitstore/splitstore_warmup.go#L89-L109